activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/TimerAuditProducerIT.java

```diff
@@ -268,5 +284,124 @@ public void shouldGetTimerCanceledEventByProcessDelete() {
+                                      startProcessEntity.getBody().getProcessDefinitionVersion()),
+                                tuple(BPMNTimerEvent.TimerEvents.TIMER_RETRIES_DECREMENTED,
+                                      "timerCatchEvent",
+                                      startProcessEntity.getBody().getBusinessKey(),
+                                      startProcessEntity.getBody().getProcessDefinitionId(),
+                                      startProcessEntity.getBody().getId(),
+                                      startProcessEntity.getBody().getProcessDefinitionKey(),
+                                      startProcessEntity.getBody().getProcessDefinitionVersion()),
+                                tuple(BPMNTimerEvent.TimerEvents.TIMER_FAILED,
+                                      "timerCatchEvent",
+                                      startProcessEntity.getBody().getBusinessKey(),
+                                      startProcessEntity.getBody().getProcessDefinitionId(),
+                                      startProcessEntity.getBody().getId(),
+                                      startProcessEntity.getBody().getProcessDefinitionKey(),
+                                      startProcessEntity.getBody().getProcessDefinitionVersion())  
+                        );
+                
+                List<CloudBPMNTimerEvent> timerEvents = receivedEvents
+                        .stream()
+                        .filter(CloudBPMNTimerEvent.class::isInstance)
+                        .map(CloudBPMNTimerEvent.class::cast)
+                        .collect(Collectors.toList());
+
+                assertThat(timerEvents)
+                        .extracting( CloudRuntimeEvent::getEventType,
+                                     CloudRuntimeEvent::getEntityId,
+                                     e -> e.getEntity().getTimerPayload().getRetries())
+                        .contains(
+                                tuple(BPMNTimerEvent.TimerEvents.TIMER_SCHEDULED,
+                                      "timerCatchEvent",
+                                      3),
+                                tuple(BPMNTimerEvent.TimerEvents.TIMER_RETRIES_DECREMENTED,
+                                      "timerCatchEvent",
+                                      2),
+                                tuple(BPMNTimerEvent.TimerEvents.TIMER_FAILED,
+                                      "timerCatchEvent",
+                                      3)  
+                );
+        });
+        
+        processInstanceRestTemplate.delete(startProcessEntity);
+    }
+    
+    public class TestActvitiEventListener implements ActivitiEventListener {
+        
+        @Override
+        public boolean isFailOnException() {
+            return false;
+        }
+        
+        @Override
+        public void onEvent(ActivitiEvent arg0) {
+            logger.info("Received Activiti Event: {}", arg0);
```

Comment:
Let's remove this for now. It may be useful for troubleshooting, but it's adding a lot of entries in the logs. We can add it again if we need to investigate some failure on that part.